clients/e-shop-demo/e-shop-frontend/src/components/checkout-total/checkout-total.component.tsx

```diff
@@ -0,0 +1,57 @@
+import { useContext, useState } from 'react';
+import { CartContext } from '../../contexts/cart.provider';
+import { UserContext } from '../../contexts/user.provider';
+import { Button } from '../../global.styles';
+import { Item } from '../../models/item.model';
+import { CheckoutHeading } from '../../pages/checkout/checkout.styles';
+import CheckoutWithIota from '../checkout-iota/checkout-iota.component';
+import MessageBox from '../message-box/message-box.component';
+import { CheckoutTotalContainer } from './checkout-total.styles';
+
+const CheckoutTotal = () => {
+	const { items, emptyCart } = useContext(CartContext);
+	const [showOrderPlaceMessage, setShowOrderPlacedMessage] = useState(false);
+	const cartHasAgeRestrictedItems = !!items.find((item: any) => item.item.ageRestricted === true);
+	const { authenticated, isVerified, setIsVerified } = useContext(UserContext);
+
+	const onCheckout = () => {
+		setIsVerified(false);
```

Comment:
is this correct? as I remember I am still verified after I'll do a checkout

Reply:
isVerified refers to the credential beeing verified. After checkout the credential is cleared so it is not longer verfied.